src/components/Card/Card.css

```diff
@@ -60,7 +60,10 @@
 
 .card .description {
   font-style: italic;
+  background: rgba(21,21,21,.01);
```

Comment:
One thing I've noticed now is that the description's background isn't visible on some monitors. I would suggest we increase the opacity just a little bit, what do you think @emmawedekind?

Reply:
Sure!

Reply:
I'll merge this now. If more changes need to be done, let's open another PR.
Let's the people enjoy this!